include/fasp_to_fenics.h

```diff
@@ -27,4 +27,8 @@ dCSRmat EigenMatrix_to_dCSRmat(const dolfin::EigenMatrix* mat_A);
 
 dvector EigenVector_to_dvector(const dolfin::EigenVector* vec_A);
 
+dolfin::EigenVector Copy_dvector_to_EigenVector(const dvector* vec_b);
```

Comment:
After thinking a bit, I actually like the implementation where we pass a pointer to ``dolfin::EigenVector`` as an input.  It adds an extra line in the main.cpp, but it is more useful for iterative solvers since we want to overwrite previous iterates. What do you think?  I think we should fix this in the other ``fasp_to_fenics`` functions too.  (Even though that was my suggestion in the first place...)